simplq/src/main/java/me/simplq/controller/model/queue/QueueDetailsResponse.java

```diff
@@ -41,4 +41,18 @@ public void addToken(me.simplq.dao.Token token) {
             token.getStatus(),
             token.getNotifiable()));
   }
+
```

Comment:
see comment below. Create a `new Token(...)` once and add to `this.tokens` or `this.deletedTokens` as per the status